packages/content/lib/database.js

```diff
@@ -200,7 +200,12 @@ class Database extends Hookable {
     // Collect data from file
     let data = []
     try {
-      data = await parser(file.data)
+      const text = await file.data
```

Comment:
That's assuming data is an `async` getter and not the parser that's the asynchronous.